src/components/mdxComponents/CtaArticleComponent.js

```diff
@@ -79,4 +86,18 @@ const CtaArticleComponent = () => {
   );
 };
 
+CtaArticleComponent.propTypes = {
```

Comment:
I have intentionally removed all the props from this component, because of the way it is being injected by Gatsby, but I quess we can keep some defaults then :)

Reply:
Yea, the reason why we need the props is because the CtaArticleComponent's text is other in the mainpage than inside the article.